server/datastore/mysql/migrations/tables/20220915153947_HostDisplayName.go

```diff
@@ -0,0 +1,40 @@
+package tables
+
+import (
+	"database/sql"
+	"github.com/pkg/errors"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20220915153947, Down_20220915153947)
+}
+
+func Up_20220915153947(tx *sql.Tx) error {
+	for _, change := range []struct{ name, sql string }{
+		{"delete index", `ALTER TABLE hosts DROP INDEX hosts_search`},
+		{"create index", `CREATE FULLTEXT INDEX hosts_search ON hosts(hostname, uuid, computer_name)`},
+		{"new table", `
+			CREATE TABLE host_display_name (
```

Comment:
Nit, but we've generally used plural form for table names, so `host_display_names` (we've been slightly inconsistent).